src/patternfly/components/Tabs/examples/Tabs.md

```diff
@@ -4,495 +4,171 @@ section: components
 ```
+
 Whenever a list of tabs is unique on the current page, it can be used in a `<nav>` element. Cases where the same set of tabs are duplicated in multiple regions on a page (e.g. cards on a dashboard) are less likely to benefit from using the `<nav>` element.
 
 ### Accessibility
-| Attribute | Applied to | Outcome |
-| -- | -- | -- |
-| `aria-label="Local"` | `nav.pf-c-tabs` | Gives the `<nav>` element an accessible name. **Required when `.pf-c-tabs` is used with `<nav>`**
-| `aria-label="Local secondary"` | `nav.pf-c-tabs.pf-m-tabs-secondary` | Gives the `<nav>` element an accessible name. **Required when `.pf-c-tabs.pf-m-tabs-secondary` is used with `<nav>`**
 
-### Usage
 | Attribute | Applied to | Outcome |
 | -- | -- | -- |
-| `.pf-c-tabs__button` | `<a>`, `<button>` | Creates a tab link/button. **Note: `<a>` should be used when there is a URL associated with the tab, and should only be used when `.pf-c-tabs` is used with `<nav>`.** **Required** |
+| `aria-label="Descriptive text"` | `nav.pf-c-tabs` | Gives the `<nav>` an accessible label. **Required when `.pf-c-tabs` is used with `<nav>`**
+| `aria-label="Descriptive text"` | `nav.pf-c-tabs.pf-m-secondary` | Gives the `<nav>` element an accessible label. **Required when `.pf-c-tabs.pf-m-secondary` is used with `<nav>`**
 
+aria-label="descriptive text"	.pf-c-inline-edit__toggle > button	Provides an accessible description for toggle button. Required
 
-## Documentation
-### Overview
-The tabs component should only be used to change content views within a page. The similar-looking but semantically different [horizontal nav component](/documentation/core/components/nav) is available for general navigation use cases.
+### Usage
 
-Tabs should be used with the [tab content component](/documentation/core/components/tabcontent).
+| Class | Applied to | Outcome |
+| -- | -- | -- |
+| `.pf-c-tabs` | `<nav>`, `<div>` | Initiates a tabs component. **Required** |
+| `.pf-c-tabs__list` | `<div>` | Initiates a tabs component list. **Required** |
+| `.pf-c-tabs__item` | `<div>` | Initiates a tabs component item. **Required** |
+| `.pf-c-tabs__button` | `<div>` | Initiates a tabs component button. **Required** |
+| `.pf-c-tabs__scroll-button` | `<div>` | Initiates a tabs component scroll button. |
+| `.pf-m-secondary` | `.pf-c-tabs` | Applies secondary styling to the tab component. |
+| `.pf-m-box` | `.pf-c-tabs` | Applies box styling to the tab component. |
+| `.pf-m-vertical` | `.pf-c-tabs` | Applies vertical styling to the tab component. |
+| `.pf-m-fill` | `.pf-c-tabs` | Applies filled styling to the tabs component. |
+| `.pf-m-current` | `.pf-c-tabs__item` | Indicates that a tab item is currently selected. |
+| `.pf-m-current` | `.pf-c-tabs__item` | Indicates that a tab item is currently selected. |
+| `.pf-m-inset-{none, sm, md, lg, xl, 2xl, 3xl}{-on-[sm, md, lg, xl, 2xl]}` | `.pf-c-tabs` | Modifies the tabs component padding/inset to visually match padding of other adjacent components. |
```

Comment:
is this a requirement for the new design?

Reply:
If the expectation is that it aligns with different, adjacent elements in different contexts, it will need insets.

Reply:
Are there designs that show use cases that this solves?

Reply:
![Screen Shot 2020-03-05 at 11 29 56 AM](https://user-images.githubusercontent.com/5385435/76002537-c20e0d80-5ed4-11ea-89e1-ba46bcdc242b.png)
 
![Screen Shot 2020-03-05 at 11 37 02 AM](https://user-images.githubusercontent.com/5385435/76003298-e0283d80-5ed5-11ea-8b71-f372e47cd1fb.png)

Reply:
`md inset`

![Screen Shot 2020-03-05 at 11 44 20 AM](https://user-images.githubusercontent.com/5385435/76003889-b4598780-5ed6-11ea-8d12-babd25d4f2aa.png)

`lg inset`

![Screen Shot 2020-03-05 at 11 44 08 AM](https://user-images.githubusercontent.com/5385435/76003925-bf141c80-5ed6-11ea-9201-5d9e18bec9dc.png)

Reply:
![Screen Shot 2020-03-05 at 11 37 02 AM](https://user-images.githubusercontent.com/5385435/76004357-5e391400-5ed7-11ea-8110-cca3f4530a52.png)